docs/Hug.md

```diff
@@ -0,0 +1,13 @@
+# `<Hug />`
+
+Use the `<Hug />` component to surround another component with a border and a banner on top.
+Pass in `props.children` to be wrapped.
+
+```jsx
+  <Hug icon='check' text={<Text.span>I hug my props.children</Text.span>} />
+```
+
+Prop | Type | Description
+---|---|---
+iconName | string | svg icon name
```

Comment:
Is this `icon` or `iconName` (I prefer the former)

Reply:
Conforming to `icon`